orte/mca/odls/default/odls_default_module.c

```diff
@@ -761,9 +761,9 @@ static int send_signal(pid_t pid, int signal)
                          signal, (long)pid));
 
     if (orte_forward_job_control) {
```

Comment:
Why did you change the last if (orte_forward_job_control) on line 395 to #if HAVE_SETPGID, but left this if(orte_forward_job_control)?  was this intentional?